ex22/main.go

```diff
@@ -1,5 +1,9 @@
 package main
 
-  func main() {
+import "fmt"
 
-  }
+func main() {
+	var a, b int64
```

Comment:
The goal of this task is to check your knowledge of the big number (>2^20) and which constructions of the language you should use for that. Please check [this](https://pkg.go.dev/math/big) package